src/lib.rs

```diff
@@ -77,3 +96,8 @@ pub struct Digest<'a, I: Implementation> {
     crc: &'a Crc<I>,
     value: I::Width,
 }
+
+// Convenient type aliases
+pub type Crc32Slice16 = Crc<Slice16<u32>>;
+pub type Crc32Bytewise = Crc<Bytewise<u32>>;
+pub type Crc32Nolookup = Crc<Nolookup<u32>>;
```

Comment:
I lean towards not having these aliases. In recommended usage, the full type is used infrequently I want to minimize API surface as much as possible. Users can always add their own aliases.

Reply:
:+1: